frontend/attestations/sbom/sbom.go

```diff
@@ -84,7 +85,9 @@ func CreateSBOMScanner(ctx context.Context, resolver sourceresolver.MetaResolver
 		}
 
 		runscan := llb.Image(scanner).Run(runOpts...)
-		runscan.AddMount("/tmp", llb.Scratch(), llb.Tmpfs())
+		if runtime.GOOS != "windows" {
```

Comment:
let's not use `runtime`, we created `integration.UnixOrWindows()` for this. I'd suggest something like this:

```go
integration.UnixOrWindows(
	runscan.AddMount("/tmp", llb.Scratch(), llb.Tmpfs()),
	runscan.AddMount("/tmp", llb.Scratch()),
)
```

Reply:
Ok, updated

Reply:
Actually this is not good. `/testutil/integration` should only be used in the tests, not in this frontend code.

Reply:
Revert and then just add Tonis' recommendation on using a cache mount. I think other than doing a TODO, let's do it within this PR...